__tests__/commands/config.js

```diff
@@ -5,13 +5,16 @@ import * as reporters from '../../src/reporters/index.js';
 import * as configCmd from '../../src/cli/commands/config.js';
 import {run as buildRun} from './_helpers.js';
 import * as fs from '../../src/util/fs.js';
+import home from 'user-home';
 
 const path = require('path');
 
+const fixturesLoc = path.join(__dirname, '..', 'fixtures', 'config');
+
 const runConfig = buildRun.bind(
   null,
   reporters.ConsoleReporter,
-  '',
+  fixturesLoc,
```

Comment:
I think this runConfig is the same for every test in the suite.
Could they be affecting each other?